packages/frontend/src/components/UserCompletionModal/index.tsx

```diff
@@ -15,20 +15,29 @@ import {
     UserCompletionModalCard,
 } from './UserCompletionModal.styles';
 
-const jobTitles = [
+function shuffleArray<T extends unknown>(arr: T[]) {
```

Comment:
the abstraction is not needed here, plus you could easily return a `string[]` to avoid the optionProps